packages/react-positioning/config/tests.js

```diff
@@ -0,0 +1,7 @@
+/** Jest test setup file. */
+
+const { configure } = require('enzyme');
+const Adapter = require('enzyme-adapter-react-16');
+
+// Configure enzyme.
+configure({ adapter: new Adapter() });
```

Comment:
Should we remove this file?

Reply:
actually forget about it, conformance tests still use enzyme...